boltstub/wiring.py

```diff
@@ -129,6 +135,101 @@ def __str__(self):
         return "[{}]:{}".format(*self)
 
 
+class RegularSocket:
+    """A socket with an already receive cached value."""
+
+    def __init__(self, _socket, _cache) -> None:
+        self._socket = _socket
+        self._cache = _cache
+
+    def __getattr__(self, item):
+        return getattr(self._socket, item)
+
+    def recv(self, __bufsize) -> bytes:
+        if (self._cache):
+            buff = self._cache
+            self._cache = None
+            return buff
+        return self._socket.recv(__bufsize)
+
+
+class WebSocket:
+    """Implementation of Websockets [rfc6455].
+
+    This implementation doesn't support extensions
+    """
+
+    def __init__(self, _socket) -> None:
+        self._socket = _socket
+
+    def __getattr__(self, item):
+        return getattr(self._socket, item)
+
+    def recv(self, __bufsize) -> bytes:
+        """Receives data from the socket.
```

Comment:
```suggestion
        """Receive data from the socket.
```

Use imperative forms for the first line of doc strings.